### DISCUSSION TITLE
禁用规则能否加个关闭开关

### DISCUSSION THREAD

**Discussion by baiyang15** [AIsouler/GKD_subscription#222] [Q&A]
不小心禁用了一个未收入软件页面的规则，能否加个关闭该条禁用规则的选项

**Comment by AIsouler** [comment]
你的意思是全局规则禁用了某个未收录软件的某个页面吗？

**Reply by baiyang15** [reply]
对的，后面在触发记录发现是可以重新启用规则（我第一个问题解决了），但在发现之前，我把那条禁用规则触发记录误删，现在还有新办法再开启那条规则了（这是新问题了）🌚，所以大佬，有办法解决吗

**Reply by baiyang15** [reply]
对了，软件是岸听音乐

**Reply by AIsouler** [reply]
全局规则>点击右边三个点>点击编辑禁用>点击上方的一个铅笔一样的按钮，这个页面可以编辑禁用的activityid，把不想禁用的删掉即可

**Reply by baiyang15** [reply]
换了个设备测试这个软件，用的大佬新的规则，发现图中广告仍然是跳不了的。我猜测是开始跳过软件更新的规则刚好关掉图中广告。![Screenshot_20240518_171134_com.kairui.discounts.cicls.jpg](https://github.com/AIsouler/GKD_subscription/assets/130110711/1d8f4a57-f581-40ee-a033-61da5074ab80)

**Reply by AIsouler** [reply]
发快照

**Reply by baiyang15** [reply]
快照记录我手欠删掉了，都清理掉了
